tests/negative/timeouts/timeouts.go

```diff
@@ -44,7 +44,7 @@ var (
 		time.Sleep(time.Second * 2)
 		// Give a config that merges ourselves and sets the timeouts to 1
 		// second (less than we wait!)
-		_, _ = w.Write([]byte(fmt.Sprintf(`{
```

Comment:
hmm, its not a blocker but it feels like the commit message is missing detail that was provided previously 

"tests/negative/timeouts: Use Fprintf instead of Write"

It would be nice to continue to reference the lint that is being fixed like before. i.e:


[
This avoids unnecessary memory allocations and resolves staticcheck
linter warnings.

Fixes lint:
\```
QF1012: Use fmt.Fprintf(...) instead of Write([]byte(fmt.Sprintf(...))) (staticcheck)
\```
]